core/dbt/contracts/graph/manifest.py

```diff
@@ -1135,6 +1135,12 @@ class WritableManifest(ArtifactMixin):
         )
     )
 
+    def __post_serialize__(self, dct):
```

Comment:
The problem we were running into with the partial parsing bug was that the config_call_dict wasn't being saved with the partial parsing manifest. So when a new schema file was added the config was being applied to the existing model node, but without the config_call_dict we lose the config from the SQL file.

There were two options to fix it, 1) save the config_call_dict or 2) every time we add a new schema file reload all of the nodes that *might* be affected. The second option would complicate partial parsing substantially and felt more inconsistent. The config_call_dict wasn't being saved to start with only because my imagination did not think about this case.  I clean it up in the WritableManifest post_serialize method because we don't really want or need it in the manifest artifact.